CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# How to contribute?
+
+## We use Conventional Commits!
+
+[![Conventional Commits](https://img.shields.io/badge/Conventional%20Commits-1.0.0-%23FE5196?logo=conventionalcommits&logoColor=white)](https://conventionalcommits.org)
+
+Conventional Commits is the specification we use to write semantically valuable commits.
+
+### Syntax
+
+```text
+<type>(<scope>): <subject>
+```
+
+- `type`: The type of your commit.
+- `scope`: The scope that is affected by your commit.
+- `subject`: A brief description of the work done in your commit.
+
+#### Supported types
+
+- `feat`
+- `style`
+- `test`
+- `docs`
+- `chore`
+- `ci`
+- `build`
+- `perf`
```

Comment:
TODO: adds a description for any type. You can reference [Conventional Commits Cheat Sheet](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index).